cmd/gator/test/test.go

```diff
@@ -72,6 +73,7 @@ func init() {
 	Cmd.Flags().StringVarP(&flagOutput, flagNameOutput, "o", "", fmt.Sprintf("Output format.  One of: %s|%s.", stringJSON, stringYAML))
 	Cmd.Flags().BoolVarP(&flagIncludeTrace, "trace", "t", false, "include a trace for the underlying Constraint Framework evaluation.")
 	Cmd.Flags().BoolVarP(&flagGatherStats, "stats", "", false, "include performance stats returned from the Constraint Framework.")
+	Cmd.Flags().BoolVarP(&flagEnableK8sCel, "prototype-enable-k8s-native-validation", "", false, "PROTOTYPE (not stable): enable the validating admission policy driver")
```

Comment:
nit: we used experimental before, do we want to use the same convention or do we think this is pre-experimental?

Reply:
I'm okay with either name.

Right now, the format for the engine hasn't really had any scrutiny (I mostly followed VAP schema). My main concern is that users know:

1) This isn't stable
2) This is intended to be the start of a discussion, not necessarily something we want to build from